src/rules/isort/split.rs

```diff
@@ -0,0 +1,40 @@
+use std::collections::BTreeSet;
+
+use crate::rules::isort::types::{ImportBlock, Importable};
+
+pub fn split_by_forced_separate<'a>(
```

Comment:
Hmm, is this not being called? Can't find the reference. Maybe I'm just tired!

Reply:
It's over here -- Github just "helpfully" collapses the big mod.rs diff if you're not looking at each commit separately:

https://github.com/charliermarsh/ruff/pull/2268/files#diff-d896c56e3618229a80f0311bdd754bc1a0c033e5c9a08f99cce34fafeda51b60R139

Reply:
D'oh, thank you...